src/VisualStudioExtension/QSharpVsix/QSharpVsix.csproj

```diff
@@ -15,8 +15,8 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Quantum.VisualStudio</RootNamespace>
     <AssemblyName>Microsoft.Quantum.VisualStudio.Extension</AssemblyName>
-    <TargetVsixContainerName>Microsoft.Quantum.Development.Kit-$(VSVSIX_VERSION).vsix</TargetVsixContainerName>
-    <LanguageServerPath>..\..\QsCompiler\LanguageServer\bin\$(Configuration)\netcoreapp3.1\publish\</LanguageServerPath>
+    <TargetVsixContainerName>Microsoft.Quantum.Development.Kit64-$(VSVSIX_VERSION).vsix</TargetVsixContainerName>
```

Comment:
Are we keeping the 64 after merging to main?

Reply:
Yes, in this case we are. This is because both extensions (VS2019 and VS2022) will remain listed concurrently in the Marketplace. They will also have different GUIDs.